app/code/Magento/Catalog/Console/ImageResizeOptions.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Catalog\Console;
+
+use Symfony\Component\Console\Input\InputOption;
+
+class ImageResizeOptions
+{
+
+    /**
+     * Key for jobs option
+     */
+    const JOBS_AMOUNT = 'jobs';
+
+    /**
+     * Key for product-offset option
+     */
+    const PRODUCT_OFFSET = 'product-offset';
+
+    /**
+     * Key for product-limit option
+     */
+    const PRODUCT_LIMIT = 'product-limit';
+
+    /**
+     * Default jobs amount
+     */
+    const DEFAULT_JOBS_AMOUNT = 0;
+
+    /**
+     * Default jobs amount
+     */
+    const DEFAULT_PRODUCT_OFFSET = 0;
+
+    /**
+     * Default jobs amount
+     */
+    const DEFAULT_PRODUCT_LIMIT = 0;
+
+    /**
+     * Image resize command options list
+     *
+     * @return array
+     */
+    public function getOptionsList()
+    {
+        return $this->getBasicOptions();
+    }
+
+    /**
+     * Basic options
+     *
+     * @return array
+     */
+    private function getBasicOptions()
```

Comment:
Do you really need a separate method which is just proxied to another public method?